python/test/ngraph/test_avg_pooling.py

```diff
@@ -0,0 +1,46 @@
+# ******************************************************************************
+# Copyright 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************
+from __future__ import print_function, division
+
+import numpy as np
+import pytest
+
+import ngraph as ng
+
+
+def test_avg_pooling_3d():
+    manager_name = pytest.config.getoption('backend', default='CPU')
+    rt = ng.runtime(manager_name=manager_name)
+
+    input = np.array(range(16), dtype=np.float32)
```

Comment:
Name input shadows built-in function name. Shouldn't use it.

Reply:
changed input -> data